lib/versioning/semver-coerced/index.spec.ts

```diff
@@ -68,15 +68,19 @@ describe('versioning/semver-coerced/index', () => {
     it('should return false for lower version in strict semver', () => {
       expect(semverCoerced.isGreaterThan('3.1.2', '4.1.0')).toBeFalsy();
     });
+
+    it('should return false if version cannot be coerced', () => {
+      expect(semverCoerced.isGreaterThan('e.e.e', '4.1.0')).toBeFalsy();
+    });
   });
 
   describe('.isLessThanRange(version, range)', () => {
     it('should return true for a lower version in strict semver', () => {
-      expect(semverCoerced.isLessThanRange('1.0.2', '~2.0')).toBeTruthy();
+      expect(semverCoerced.isLessThanRange?.('1.0.2', '~2.0')).toBeTruthy();
```

Comment:
Why is this necessary? Isn't it always defined?

Reply:
It's an optional function
https://github.com/renovatebot/renovate/blob/74d7691befa635c1220d8569936273bfb950e748/lib/versioning/types.ts#L26